src/libraries/Hooks.sol

```diff
@@ -184,11 +257,28 @@ library Hooks {
         IHooks self,
         PoolKey memory key,
         IPoolManager.SwapParams memory params,
-        BalanceDelta delta,
-        bytes calldata hookData
-    ) internal noSelfCall(self) {
+        BalanceDelta swapDelta,
+        bytes calldata hookData,
+        int128 hookDeltaSpecified
```

Comment:
We use input someplaces vs specified/unspecified other places. Can we make consistent? Also happy for this to be in followup to just make some of this more uniform.

ie hookDeltaOnInput & hookDeltaOnOutput maybe are more straightforward.. otherwise would just change any occurences of inputCurrency to specifiedCurrency

Reply:
oh also maybe we can swap these two params around to match the other func sigs where the hookData is the last param